packages/ckeditor5-font/docs/features/font.md

```diff
@@ -355,6 +355,35 @@ ClassicEditor
 	.catch( /* ... */ );
 ```
 
+### Color picker
+
+Colors from outside of the preconfigured palette can be set using the "Color picker" option available at the bottom of the color selection dropdown.
+
+Color picker applies colors in the **HSL** format as it is default for the font color feature. This can be changed using the {@link module:ui/colorpicker/utils~ColorPickerConfig#Format `config.fontColor.colorPicker.format`} option. Available color formats are defined in the {@link module:ui/colorpicker/utils~ColorPickerOutputFormat} type. Note that this change will not affect the color input &ndash; it always accepts only values given in the `hex` format (with or without the `#` sign at the beginning).
```

Comment:
> (with or without the `#` sign at the beginning)
To my mind this information is not necessary for developers because it changes nothing. This info is just nice to know for actual users which will use that but not for developers. It just my mind, so it could actually stay.

Reply:
I think actual users may also sometimes read the documentation, or ask dev to do it for them :stuck\_out\_tongue\_winking\_eye: so let's keep it, it's just a few extra words.